src/pages/api/register/create_account.ts

```diff
@@ -108,7 +146,10 @@ export default async function handler(
       displayName,
       twitterUsername,
```

Comment:
We don't need to store these anymore right? Since we're defaulting to only consent based sharing each time?

Reply:
All of the usernames and bio too?

Reply:
This is a good point, let me fix this

Reply:
Fixed! Removed socials - great catch